features/scv/jceks-impl/src/main/java/org/opennms/features/scv/jceks/JCEKSSecureCredentialsVault.java

```diff
@@ -102,29 +105,36 @@ public JCEKSSecureCredentialsVault(String keystoreFile, String password, byte[]
                     m_keystore.load(is, m_password);
                 }
             }
+            loadCredentials();
         } catch (KeyStoreException | NoSuchAlgorithmException | CertificateException | IOException e) {
             throw Throwables.propagate(e);
         }
     }
-    
-    @Override
-    public Credentials getCredentials(String alias) {
+
+    private void loadCredentials() {
```

Comment:
This may increase startup time, should be moved to first call in `getCredentials`